syntax = "proto2";

package ZilliqaMessage;

import "ZilliqaMessage.proto";


message CreateTransactionRequest
{
    required ProtoTransaction tx = 1;
}

